rest/taskana-rest-spring/src/main/java/pro/taskana/rest/resource/links/PageLinksAspect.java

```diff
@@ -26,32 +27,35 @@
 @Aspect
 public class PageLinksAspect {
 
+  @SuppressWarnings("unchecked")
```

Comment:
Do you want to try again and remove this supresswarning? Or should we leave it here for now?